src/sentry/models/projectsdk.py

```diff
@@ -0,0 +1,45 @@
+from collections.abc import Sequence
+from enum import Enum
+
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import (
+    BoundedBigIntegerField,
+    BoundedIntegerField,
+    Model,
+    region_silo_model,
+    sane_repr,
+)
+from sentry.utils.snowflake import snowflake_id_model
+
+
+class EventType(Enum):
+    PROFILE = 0
+    PROFILE_CHUNK = 1
+
+    @classmethod
+    def as_choices(cls) -> Sequence[tuple[int, str]]:
+        return (
+            (cls.PROFILE.value, "profile"),
+            (cls.PROFILE_CHUNK.value, "profile_chunk"),
+        )
+
+
+@snowflake_id_model
+@region_silo_model
+class ProjectSDK(Model):
+    __relocation_scope__ = RelocationScope.Organization
+
+    date_updated = models.DateTimeField(auto_now=True)
+
+    project_id = BoundedBigIntegerField(db_index=True)
```

Comment:
You could use a FlexibleForeignKey here. Without a foreign key, you'll need to add this model to project deletion logic.